### ISSUE TITLE
setup.py extract_messages stopped working with chameleon

### ISSUE THREAD

**Issue by Estartu** [malthe/chameleon#243]
I use chameleon and pyramid in a small project with i18n tags for translation. 
I did the project a year ago and back than i simply run python setup.py extract_messages to get all the translation messages from *.py an *.pt files. 
message_extractor setting in the setup.py was 
{'.': [('**.py', 'python', None),('**.pt', 'tales', None)]}

I started to do some extending this week and first upgraded all used packages to the most recent version. Everything work fine but extract_messages stopped working on .pt files. 

pot-create still works for .pt file so it's not really pressing at the moment. 

I've done some debugging and the setuptools search for extract_messages entrypoints in every installed distribution. Python and javascript they handle themselves but any other extractor has to come from the package installed. As far as i can see only mako has such a entrypoint defined.

**Comment by malthe** [comment]
Perhaps you're missing a dependency? Chameleon itself has – as far as I recall – no such functionality built-in.

**Comment by Estartu** [comment]
yeah lingua does this for chameleon at the moment so pot-create (comes with linuga) works for now. 

But it looks like setuptools tries to establish a new generic way. I think that's a really good idea to let the guys who know the syntax do the extracting.

So chameleon should provide the extractor for i18n messages in pagetemplates.
